cluster-autoscaler/main.go

```diff
@@ -524,7 +527,7 @@ func buildAutoscaler(debuggingSnapshotter debuggingsnapshot.DebuggingSnapshotter
 			return nil, err
 		}
 		opts.LoopStartNotifier = loopstart.NewObserversList([]loopstart.Observer{provreqProcesor})
-		injector, err := provreq.NewProvisioningRequestPodsInjector(restConfig)
+		injector, err := provreq.NewProvisioningRequestPodsInjector(restConfig, autoscalingOptions.LastProvisioningRequestSeenTime)
```

Comment:
I do not  think the value here should be null. The second arg is a pointer to the lastProvisioningRequestSeenTime so that it can be modified by the injector when it, it marks a ProvisioniningRequest as Accepted or Failed.

Reply:
I meant that injector is not relied on LastProvisioningRequestSeenTime value and so LastProvisioningRequestSeenTime could be set directly in injector.
As Karol mentioned AutoscalingOption is used to store CA configs and LastProvisioningRequestSeenTime is CA config parameter.

Reply:
Got it. Updated accordingly.